index.js

```diff
@@ -0,0 +1,60 @@
+let opcao_select_diamante = document.getElementById('select-diamante');
+let divMostraDiamante = document.getElementById('mostra-diamante');
+let emBranco = '&nbsp'.repeat(3);
+let diamante = [];
+let meioDiamante;
+let texto = '';
+
+geraSelect();
+
+function mostrarDiamante(valor){
+    
+    for(let i=0;i<meioDiamante;i++)
+    {   
+        for(let j=0;j<valor;j++){
+            
+            if(meioDiamante-1 == j)
+            { 
+                let charCode = String.fromCharCode(i+65);
+                diamante[j-i] = charCode
+                diamante[j+i] = charCode
+            }           
+        }
+        texto+=diamante.join('')+'<br>';
+        diamante.fill(emBranco);      
+    }
+    
+    for(i= opcao_select_diamante.value -1;i>=1;i--)
+    { 
+        for(j=0;j<=valor;j++)
+            {
+                if(meioDiamante-1 == j)
+                { 
+                    let charCode = String.fromCharCode(i+64);
+                    diamante[(j-i)+1] = charCode;
+                    diamante[(j+i)-1] = charCode;
+                }
+            }
+        texto+=diamante.join('')+'<br>';
+        diamante.fill(emBranco);
+    }
+    return divMostraDiamante.innerHTML=texto;  
+}
+
+function geraSelect(){
+    for(let i = 1;i<=26;i++){
```

Comment:
```suggestion
    let totalLetrasNoAlfabeto = 26;
    for(let i = 1;i<=totalLetrasNoAlfabeto;i++){
```

Quem lê seu código, não sabe o que o número 26 representa.